src/Couchbase.Lite.Shared/API/Sync/ReplicatorStatusChangedEventArgs.cs

```diff
@@ -44,4 +45,28 @@ internal ReplicatorStatusChangedEventArgs(ReplicatorStatus status)
 
         #endregion
     }
+
+    /// <summary>
+    /// Event arguments for the <see cref="Replicator.AddDocumentEndedListener(EventHandler{DocumentReplicatedEventArgs})" /> event
```

Comment:
Is this still correct `Replicator.AddDocumentEndedListener` ?  If not fix this one and one more here.

Reply:
fixed